src/components/Atoms/Link/Link.md

```diff
@@ -1,33 +1,61 @@
 ```js
 <div>
   <div style={{ display: 'block', marginBottom: '10px' }}>
-    <Link
-      color="black"
-      href="#anchor"
-      target="self"
-      inline={false}
-      wrapper={true}
-    >
-      <a href="#">Link as button</a>
+    <Link href="#anchor" target="blank" linktype="standard">
+      Link standard
     </Link>
   </div>
   <div style={{ display: 'block', marginBottom: '10px' }}>
-    <Link color="red" href="/test" target="self">
-      Link as button
+    <Link href="#anchor" linktype="bold">
```

Comment:
Should this be `weight` instead type.

Reply:
weight could also be done via `Text`.

Reply:
yeah that's true. It used to be one of the link types we had and if you'd use the bold class you'd get a bold link you see. I'll remove it and just keep the standard and standard_white link styles 👍